### DISCUSSION TITLE
​Edit internal chain

### DISCUSSION THREAD

**Discussion by shanyurc** [toeverything/AFFiNE#6494] [General]
### Description

Whether it can increase the function to allow the editor's internal chain name

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by EYHN** [comment]
Can you describe this feature in detail? I don't quite understand

**Reply by shanyurc** [reply]
![image](https://github.com/toeverything/AFFiNE/assets/58464977/3e191407-abab-4067-bbf5-cfe7ed29cd92)
I hope to modify this link name

**Comment by shanyurc** [comment]
![image](https://github.com/toeverything/AFFiNE/assets/58464977/32dffa7a-cbf4-4fd4-884a-e9b481a80d98)
I hope to modify this link name
